xml/System.Data/DataRowView.xml

```diff
@@ -539,7 +539,17 @@
         <summary>Gets a value indicating whether the current <see cref="T:System.Data.DataRowView" /> is identical to the specified object.</summary>
         <returns>
           <see langword="true" /> if <paramref name="object" /> is a <see cref="T:System.Data.DataRowView" /> and it returns the same row as the current <see cref="T:System.Data.DataRowView" />; otherwise <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Checks for same reference instead of equivalent <xref:System.Data.DataRowView.DataView> or <xref:System.Data.DataRowView.Row>.
```

Comment:
```suggestion
This method checks for reference equality instead of checking whether <xref:System.Data.DataRowView.DataView> or <xref:System.Data.DataRowView.Row> are equivalent.
```